subsys/bluetooth/host/Kconfig

```diff
@@ -732,6 +732,14 @@ config BT_TESTING
 	  This option enables custom Bluetooth testing interface.
 	  Shall only be used for testing purposes.
 
+config BT_CONN_DISABLE_SECURITY
+	bool "Disable security"
+	depends on BT_TESTING
+	help
+	  This option disables security checks for incoming requests enabling
+	  to test accessing GATT attributes and L2CAP channels that would
+	  otherwise require encryption/authentication in order to be accessed.
+
```

Comment:
Can you please add a paragraph below like:

```
  WARNING: This option enables anyone to snoop on-air traffic.
  Use of this feature in production is strongly discouraged.
```

Reply:
Done.